tests/sparsezoo/models/classification/test_efficientnet.py

```diff
@@ -39,15 +42,16 @@ def test_efficientnet_b0(
     sparse_target,
 ):
     model_constructor(
-        efficientnet_b0,
-        download,
-        framework,
-        repo,
-        dataset,
-        training_scheme,
-        sparse_name,
-        sparse_category,
-        sparse_target,
+        constructor_function=efficientnet_b0,
+        download=download,
+        framework=framework,
+        repo=repo,
+        dataset=dataset,
+        training_scheme=training_scheme,
+        sparse_name=sparse_name,
+        sparse_category=sparse_category,
+        sparse_target=sparse_target,
+        test_name=os.path.join(str(os.path.basename(__file__)), str(datetime.now())),
```

Comment:
nit: `os.path.basename` returns `str` already
```suggestion
        test_name=os.path.join(os.path.basename(__file__), str(datetime.now())),
```